src/Nethermind/Nethermind.Evm/Instruction.cs

```diff
@@ -84,6 +84,8 @@ public enum Instruction : byte
         MSTORE8 = 0x53,
         SLOAD = 0x54,
         SSTORE = 0x55,
+        TLOAD = 0xb3,
+        TSTORE = 0xb4,
```

Comment:
Lets keep values here in order of code, can add <see> comments to link to S versions